roles/confluent.control_center/defaults/main.yml

```diff
@@ -1,6 +1,11 @@
 ---
 # More control_center variables are defined in the confluent.variables role
 
+### Boolean to reconfigure Kafka's logging with RollingFileAppender and log cleanup
+control_center_custom_log4j: "{{ custom_log4j }}"
+
+control_center_log4j_root_logger: "INFO, main"
```

Comment:
this ones missing stdout

Reply:
by default c3 doesn't include `stdout` in `$CONFLUENT_HOME/etc/confluent-control-center/log4j-rolling.properties`. Is it ok to keep it align with this file?

Reply:
Ya ok thats fine then. Did you get all the `<component>_log4j_root_logger` out of the log4j files that ship w the packages?